### DISCUSSION TITLE
Stargazer on mobile

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#156] [Questions]
Hi,
so i just updated to the newest Version and tried out the new Theme.
Looks very cool, but on mobile Devices it's not centered:
![image](https://user-images.githubusercontent.com/47309435/189065317-d20c0ef8-4410-4e98-8ac7-031e17e1f9af.jpeg)

**Comment by JulianPrieber** [comment]
I released a quick fix to https://github.com/JulianPrieber/stargazer.
This should do it for most resolutions.

**Reply by root9191** [reply]
Thanks for your fast Support 🤗
